Basic/DistributedAuthoritySocialHub/Assets/Editor/Tutorials/Tutorial3_SessionOwnerPromotion/1-SessionOwnerPromotion.asset

```diff
@@ -0,0 +1,188 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: ff771ccdf4150419d9ff4d342b069aae, type: 3}
+  m_Name: 1-SessionOwnerPromotion
+  m_EditorClassIdentifier: 
+  Title:
+    m_Untranslated: Session Owner Promotion With Distributed Authority
+  m_Paragraphs:
+    m_Items:
+    - m_Type: 6
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: 
+      m_CodeSample: 
+      m_PostInstructionImage: {fileID: 0}
+      m_Tutorial: {fileID: 0}
+      m_Image: {fileID: 0}
+      m_VideoUrl: 
+      m_Video: {fileID: 0}
+      m_CriteriaCompletion: 0
+      m_Criteria:
+        m_Items: []
+      m_MaskingSettings:
+        m_MaskingEnabled: 1
+        m_UnmaskedViews: []
+      m_Summary: 
+      m_Description: 
+      m_InstructionBoxTitle: 
+      m_InstructionText: 
+      m_TutorialButtonText: 
+    - m_Type: 0
+      Title:
+        m_Untranslated: Page title
+      Text:
+        m_Untranslated: 'Netcode for GameObjects v2.0+ supports building games with
+          a <b>distributed authority</b> topology, providing more options for ownership
+          and authority over NetworkObjects.
+
+
+          Distributed authority provides
+          additional permissions to manage object distribution while also incorporating
+          the concept of a session owner. The session owner is given host-like permissions
+          to handle global game state-related tasks that a typical client-server
+          host handles.
+
+
+          As long as global game states are persisted via
```

Comment:
This global game state bit adds confusion I think, it's not something the user has to actively do. If you think we should still call it out, please add a note after this section.